libsel4/sel4_arch_include/x86_64/interfaces/sel4arch.xml

```diff
@@ -29,14 +29,21 @@
         <member name="gs_base"/>
     </struct>
 
-    <interface name="seL4_X86_PDPT" manual_name="PDPT">
+    <interface name="seL4_X86_PDPT" manual_name="PDPT" cap_description="Capability to the PDPT being operated on.">
         <method id="X86PDPTMap" name="Map">
+            <brief>
+                Map a page directory page table.
+            </brief>
             <description>
-                TODO
+               <docref>See <autoref label="ch:vspace"/></docref>
             </description>
-            <param dir="in" name="pml4" type="seL4_X64_PML4"/>
-            <param dir="in" name="vaddr" type="seL4_Word"/>
-            <param dir="in" name="attr" type="seL4_X86_VMAttributes"/>
+            <param dir="in" name="pml4" type="seL4_X64_PML4" description="Capability to the PDPT which will contain the mapping."/>
```

Comment:
```suggestion
            <param dir="in" name="pml4" type="seL4_X64_PML4" description="Capability to the VSpace which will contain the mapping."/>
```
It's probably a `seL4_X64_PML4Object`, but it seems all the weird objects are just called VSpace in x86 documentation.